package.json

```diff
@@ -237,31 +237,31 @@
       "version": "0.1.2"
     },
     {
-      "hash": "QmfDWfnZZiwHSD58FE2PveLQhZZhZjbuvo1TU1Zu4P9Hd3",
+      "hash": "QmTHyAbD9KzGrseLNzmEoNkVxA8F2h7LQG2iV6uhBqs6kX",
       "name": "go-libp2p-kad-dht",
-      "version": "2.5.4"
+      "version": "2.5.5"
     },
     {
       "author": "whyrusleeping",
-      "hash": "QmaP28E6WHN9sqgqh71YrMMGVZqWvunD3tDJyVNLX6c2zZ",
+      "hash": "QmV6pzBTpFEyoPCFWndHW1y5EM9bpK7j7SELSXyvfmqW8u",
       "name": "hang-fds",
-      "version": "0.0.4"
+      "version": "0.0.5"
     },
     {
       "author": "whyrusleeping",
-      "hash": "QmUpeULWfmtsgCnfuRN3BHsfhHvBxNphoYh4La4CMxGt2Z",
+      "hash": "QmZdsQf8BiCpAj61nz9NgqVeRUkw9vATvCs7UHFTxoUMDb",
       "name": "floodsub",
-      "version": "0.8.14"
+      "version": "0.8.15"
     },
     {
       "author": "whyrusleeping",
-      "hash": "QmPAKbSsgEX5B6fpmxa61jXYnoWzZr5sNafd3qgPiSH8Uv",
+      "hash": "QmYNyRZJBUYPNrLszFmrBrPJbsBh2vMsefz5gnDpB5M1P6",
       "name": "go-ipld-format",
-      "version": "0.4.10"
+      "version": "0.5.0"
     },
     {
       "author": "whyrusleeping",
-      "hash": "Qmcdid3XrCxcoNQUqZKiiKtM7JXxtyipU3izyRqwjFbVWw",
+      "hash": "QmTJ1WLpMnrPqhBuPVNSNbeHJgiY3d2MpcWkLVyvg4xB2H",
       "name": "go-ipld-cbor",
       "version": "1.2.5"
```

Comment:
Update to QmemYymP73eVdTUUMZEiSpiHeZQKNJdT5dP2iuHssZh1sR. Sorry, I didn't bump the version number when I released the last version of go-ipld-cbor :disappointed:.